main.go

```diff
@@ -73,6 +80,41 @@ func runDocker2ACI(arg string, flagNoSquash bool, flagImage string, flagDebug bo
 	return nil
 }
 
+func printConvertedVolumes(aciPaths []string) error {
```

Comment:
I would pass just a string instead of an array. Or at least document why we are using only last element here.